lib/wasix/src/syscalls/wasi/fd_read.rs

```diff
@@ -44,6 +44,7 @@ pub fn fd_read<M: MemorySize>(
         fd_entry.offset.load(Ordering::Acquire) as usize
     };
 
```

Comment:
We probably also would want to include `sock_send` and `sock_recv`?

Reply:
Those syscalls are only active when the process is active (i.e. something is being actively done and we do not want to slow down) - thus adding the overhead of the check is redundant (although likely unmeasurable)